.azure/azure-pipelines.ci.yml

```diff
@@ -602,7 +602,7 @@ stages:
       platform: windows
       tls: schannel
       logProfile: Full.Light
-      extraTestArgs: -Filter -*ValidateConfiguration:*ValidAlpnLengths:*ResumeRejection*:*ClientCertificate*:*LoadBalanced*
+      extraTestArgs: -Filter -*ValidateConfiguration:*ValidAlpnLengths:*ResumeRejection*:*ClientCertificate*:*LoadBalanced*:TlsTest.InProc*
```

Comment:
So they don't run on WS2022? Only Win-Latest (internal CI)?

Reply:
This makes me thing we need to onboard a run with internal machines for public CI.

Reply:
I don't understand the second comment. I think this only works on Win-Latest. I'll remove for the next commit and see if it fails CI.

Reply:
yep it fails; only works on Win-Latest. Let's chat tomrrow about the onboarding comment.

Reply:
Ok. We can follow up with possibly onboarding an internal CI run triggered from GitHub (in the future).